### PR TITLE
Integrating Latest Aztec

### PR DESCRIPTION
**Pull request by jleandroperez**
### Details:
- New Header Style Picker
- Wired the new Format Bar

**Notes:**
- *OptionsTableView* is a direct port from what we've got in Aztec. Definitely happy to implement a custom / tailored solution, but for Mark I, let's bring in what we currently have in the Aztec Example!
- Mapped the Podfile to the latest commit. I'll update that to a stable release, when possible!

Needs review: @diegoreymendez @astralbodies @SergioEstevao 
Thanks in advance gentlemen!

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -51,7 +51,7 @@ abstract_target 'WordPress_Base' do
     pod 'WPMediaPicker', '0.13'
     pod 'WordPress-iOS-Editor', '1.9.0'
     pod 'WordPressCom-Analytics-iOS', '0.1.23'
-    pod 'WordPress-Aztec-iOS', '0.5a6'
+    pod 'WordPress-Aztec-iOS', :git => 'https://github.com/wordpress-mobile/WordPress-Aztec-iOS.git', :commit => '5ead13b91446d00135d78f01479d36dac6c84f45'
```
Comment (astralbodies) [inline Podfile]: Lets not use commit hashes any more - go ahead and version the pod.

Comment (jleandroperez) [inline Podfile]: @diegoreymendez can we release a new Aztec version?

Comment (diegoreymendez) [inline Podfile]: @astralbodies - Are we sure we want to version a full version for each integration?  That feels like a lot of overhead considering the library is being actively developed alongside WPiOS.

By integrating frequently (without the need to version each time) we're reducing the time it takes to solve dependencies between issues.

**Comment by SergioEstevao** [comment]
@astralbodies @diegoreymendez this is ready for another look.

I follow @astralbodies recommendation and update the Aztec pod using a tag. This tag can later be converted to an official cocoa pod version.

With this update I also added:

 - better detection of taps on attachments;
 - added the option to insert horizontal rulers on the toolbar;
 - more bug fixes;

**Comment by astralbodies** [comment]
This is aimed at `develop` - is that what you wanted?

**Comment by SergioEstevao** [comment]
@astralbodies it was what @jleandroperez targeted initially so let's go like that for this. I will cherry pick changes to 7.2 on another PR.

**Comment by astralbodies** [comment]
:shipit: into develop

**Comment by jleandroperez** [comment]
Thanks everyone!

**Pull request merged**